root/static/scripts/release-editor/edits.js

```diff
@@ -608,7 +608,7 @@ releaseEditor.orderedEditSubmissions = [
     callback: function (release, edits) {
       var edit = edits[0];
 
-      if (edit.edit_type == MB.edit.TYPES.EDIT_RELEASEGROUP_CREATE) {
+      if (edit?.edit_type == MB.edit.TYPES.EDIT_RELEASEGROUP_CREATE) {
```

Comment:
This looks fine, but can we add a short comment specifying when `edit` is undefined so we don't forget?